garrysmod/html/js/menu/control.Servers.js

```diff
@@ -230,10 +230,7 @@ function AddServer( type, id, ping, name, desc, map, players, maxplayers, botpla
 		workshopid:		workshopid
 	};
 
-	data.hasmap = DoWeHaveMap( data.map );

```

Comment:
Are you sure? It's data.hasmap and at a first glance, it shows that the sv variable isn't the data variable but I could be wrong.